api/transfer.go

```diff
@@ -85,11 +87,18 @@ func (w Wrapper) UpdateTransfer(ctx echo.Context, transferID string) error {
 	if err != nil {
 		return err
 	}
-	transfer, err := w.TransferSenderRepo.Update(ctx.Request().Context(), cid, transferID, func(t *domain.Transfer) (*domain.Transfer, error) {
+
+	_, err = w.TransferSenderRepo.Update(ctx.Request().Context(), cid, transferID, func(t *domain.Transfer) (*domain.Transfer, error) {
 		//t.Description = updateRequest.Description
 		t.TransferDate = updateRequest.TransferDate
 		return t, nil
 	})
+
+	transfer, err := w.TransferSenderService.GetTransferByID(ctx.Request().Context(), cid, transferID)
```

Comment:
What does this fix, and why doest the Update method not return a defferent version?

Reply:
This calls the `TransferSenderService` which retrieves additional information while the update method calls the `TransferSenderRepository` which gets the data directly from the database but only returns a limited set of data.